integration-tests/src/test/resources/setupenv.sh

```diff
@@ -179,7 +179,15 @@ if [ "$SHARED_CLUSTER" = "true" ]; then
 	fi
     	
   #fi
-  setup_shared_cluster
+  helm version --short --client | grep v2
+  [[ $? == 0 ]] && HELM_VERSION=V2
+  [[ $? == 1 ]] && HELM_VERSION=V3
+  if [ "$HELM_VERSION" == "V2" ]; then
```

Comment:
I'd recommend failing the test if you find Helm 2.x.  This will help us upgrade the clusters quickly.

Reply:
done